### PR TITLE
Temp fix npe

### PR DESCRIPTION
**Pull request by Emibergo02**
[00:33:03 ERROR]: Could not pass event PlayerInteractEvent to Oraxen v1.154.2
java.lang.NullPointerException: Cannot invoke "io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.DisplayEntityProperties.getDisplayTransform()" because "<parameter4>" is null
        at io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.FurnitureMechanic.setItemDisplayData(FurnitureMechanic.java:415) ~[oraxen-1.154.2.jar:?]
        at io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.FurnitureMechanic.setEntityData(FurnitureMechanic.java:377) ~[oraxen-1.154.2.jar:?]
        at io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.FurnitureMechanic.lambda$place$3(FurnitureMechanic.java:345) ~[oraxen-1.154.2.jar:?]
        at org.bukkit.craftbukkit.v1_19_R3.CraftRegionAccessor.addEntity(CraftRegionAccessor.java:545) ~[purpur-1.19.4.jar:git-Purpur-1949]
        at org.bukkit.craftbukkit.v1_19_R3.CraftRegionAccessor.spawn(CraftRegionAccessor.java:524) ~[purpur-1.19.4.jar:git-Purpur-1949]
        at org.bukkit.craftbukkit.v1_19_R3.CraftRegionAccessor.spawn(CraftRegionAccessor.java:518) ~[purpur-1.19.4.jar:git-Purpur-1949]
        at org.bukkit.craftbukkit.v1_19_R3.CraftRegionAccessor.spawn(CraftRegionAccessor.java:509) ~[purpur-1.19.4.jar:git-Purpur-1949]
        at io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.FurnitureMechanic.place(FurnitureMechanic.java:344) ~[oraxen-1.154.2.jar:?]
        at io.th0rgal.oraxen.mechanics.provided.gameplay.furniture.FurnitureListener.onFurniturePlace(FurnitureListener.java:171) ~[oraxen-1.154.2.jar:?]

### CODE REVIEW

**Comment by Boy0000** [comment]
closing this as this is not the solution here
ill be including a fix for this weeks update

tldr; if displayConfigSection == null properties = new DisplayEntityProperties() which just has all values set to a default value
